datasets/geonet.yaml

```diff
@@ -0,0 +1,38 @@
+Name: GeoNet Aotearoa New Zealand Data
+Description: GeoNet provides geological hazard information for Aotearoa New Zealand. This dataset contains data and products recorded by the GeoNet sensor network. The dataset currently include GNSS data and additional datasets will be added in the near future.
+  GNSS (Global Navigation Satellite System) data include raw data in proprietary and Receiver Independent Exchange Format (RINEX) and local tie-in survey conducted during equipment changes '(https://www.geonet.org.nz/data/types/geodetic)'.
+Documentation: https://github.com/GeoNet/data-tutorials/tree/main/AWS_Open_Data
+Contact: For questions regarding data content or quality, please visit '[the GeoNet site](https://www.geonet.org.nz/)]', contact info@geonet.org.nz or use GitHub issue on [GeoNet/help](https://github.com/GeoNet/help).
+ManagedBy: "[GeoNet](https://www.geonet.org.nz/)"
+UpdateFrequency: Daily for majority of datasets.
+Tags:
+  - aws-pds
+  - earth science
+  - geophysics
+  - geological hazard
+  - natural hazard monitoring
+  - tectonics
+  - geodesy
+License: GeoNet data are made available free of charge under '[CC3 licence](https://creativecommons.org/licenses/by/3.0/nz/)' to facilitate research and risk assessment. Please acknowledge the GeoNet programme and its sponsors when using the data, see '[GeoNet Data Policy](https://www.geonet.org.nz/policy)'. 
+Resources:
+  - Description: 
+    ARN: arn:aws:s3:::geonet-open-data
+    Region: ap-southeast-2
+    Type: S3 Bucket
+    Explore:
+    - '[Browse Bucket](https://geonet-open-data.s3.amazonaws.com/index.html)'

```

Comment:
I tried to access this but response was "The specified key does not exist"

Reply:
yeah it's not automatic, you'll want to enable static website hosting, see https://docs.aws.amazon.com/AmazonS3/latest/userguide/IndexDocumentSupport.html

Reply:
@elidana this might be of interest: https://github.com/awslabs/aws-js-s3-explorer

Reply:
oooh super interesting, thanks!! I'll share with the team here

Reply:
I have deleted the index entry for now, and will reintroduce in case we decide to use this feature. thanks a lot for sharing the links @jeffejefe , those are quite useful